backend/src/lib.rs

```diff
@@ -40,6 +41,10 @@ pub fn router(pool: SqlitePool) -> Result<Router, Box<dyn Error>> {
     let election_routes = Router::new()
         .route("/", get(election::election_list))
         .route("/:election_id", get(election::election_details))
+        .route(
+            "/:election_id/download-pdf",
```

Comment:
We use snake case for other paths, so:
```suggestion
            "/:election_id/download_pdf",
```

Reply:
[58f0f1e](https://github.com/kiesraad/abacus/commit/58f0f1eee51de154409f9e2784ca8d2e54632c57) addresses this